src/test/java/com/exasol/adapter/dynamodb/ExasolTestInterface.java

```diff
@@ -100,15 +113,22 @@ public void createConnection(final String name, final String to, final String us
      * @throws SQLException on SQL error
      */
     public void createAdapterScript() throws SQLException {
+        setScriptOutputAddress();
+
         createTestSchema(ADAPTER_SCHEMA);
         final StringBuilder statementBuilder = new StringBuilder(
                 "CREATE OR REPLACE JAVA ADAPTER SCRIPT " + ADAPTER_SCHEMA + "." + DYNAMODB_ADAPTER + " AS\n");
         final String hostIp = getTestHostIpAddress();
 
-        if (hostIp != null && !isNoDebugSystemPropertySet()) {
-            // noinspection SpellCheckingInspection
-            statementBuilder.append("  %jvmoption -agentlib:jdwp=transport=dt_socket,server=n,address=").append(hostIp)
-                    .append(":8000,suspend=y;\n");
+        if (hostIp != null) {
+            String jvmOptions = "-javaagent:/buckets/bfsdefault/default/" + JACOCO_JAR_NAME
```

Comment:
Make final and use `?` operator or use `StringBuilder`.

Reply:
I dont like to use the ? operator here. Its not an alternative. Instead something is added conditionally. I think using an if its much more readable here. How about the solution with the StringBuilder?

Reply:
Yes, that is also perfectly fine.